int64[] row_coords
int64[] column_coords
---
bool success
